docs/TRANSLATIONS.md

```diff
@@ -1,115 +1,121 @@
 # Translations
 
-## Help us to review/proofread translations
+Translations are managed through [Weblate][weblate]. Please [contribute][contribute] translations via the [Weblate][weblate], and the system and maintainers will handle the rest.
 
-You can join our [GitHub translation teams](https://github.com/orgs/organicmaps/teams/translations/teams),
-so any contributor can tag all teams (or a specific language team) to get help with the review.
+## Components
 
-Please respond in the relevant [GitHub discussion](https://github.com/orgs/organicmaps/discussions/8538), or let us know at hello@organicmaps.app
+The project consists of multiple components, each with its own translation files.
 
-## Contribute translations directly
+| Weblate Component                                   | Description                                                | Translation Files                                                                                        |
+| --------------------------------------------------- | ---------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| [Android][android_weblate]                          | UI strings                                                 | [android/app/src/main/res/values\*/strings.xml][android_git] ([en][android_git_en])                      |
+| [Android Type Strings][android_typestrings_weblate] | OpenStreetMap Types                                        | [android/app/src/main/res/values\*/type_strings.xml][android_git] ([en][android_type_strings_git_en])    |
```

Comment:
```suggestion
| [Android feature types][android_typestrings_weblate] | Map feature types                                        | [android/app/src/main/res/values\*/type_strings.xml][android_git] ([en][android_type_strings_git_en])    |
```

They're not 1:1 mapping of OSM tags/types, its better to avoid possible confusion